src/helperFunctions/Filter/inititalFilter.js

```diff
@@ -0,0 +1,14 @@
+const initialState = {
+  sortBy: null,
+  categories: {
+    Fiction: false,
+    Non_fiction: false,
+    Philosophy: false,
+    Family_and_relationship: false,
+    Self_help: false,
+  },
```

Comment:
if u get time think about how u would get categories from API instead of hard coding it here